rest/data/patch.go

```diff
@@ -197,35 +197,56 @@ func AbortPatchesFromPullRequest(ctx context.Context, event *github.PullRequestE
 	return nil
 }
 
+type RawPatch struct {
+	Patch      RawModule   `json:"patch"`
+	RawModules []RawModule `json:"raw_modules"`
+}
+type RawModule struct {
+	Name    string `json:"name"`
+	Diff    string `json:"diff"`
+	Githash string `json:"githash"`
+}
+
 // GetPatchRawPatches fetches the raw patches for a patch
```

Comment:
nit: I know this line wasn't changed, but period at the end of doc comment sentence.

Reply:
I get it, but the line is the same as it was before 😅

Reply:
I think it's now "// GetRawPatches fetches the raw patches for a patch."